app/views/layouts/application.html.erb

```diff
@@ -10,6 +10,13 @@
   </head>
 
   <body>
+    <% flash.each do |type, msg| %>
+      <br>
+      <div class=<%=type%> >
+        <%= msg %>
+      </div>
+    <% end %>
```

Comment:
Glad you added this, forgot about it and it will be great for happy and sad path testing.